### DISCUSSION TITLE
Logging GSLB changing state by LUA records

### DISCUSSION THREAD

**Discussion by SkySai1** [PowerDNS/pdns#14304] [Q&A] [answered]
Hello everyone!
Is there some way to logging change event of address availability on LUA records? 
If im using LUA A "ifportup(443, {'1.1.1.1', 2.2.2.2}) and '1.1.1.1' was changed to 'unavailable/available' is there a way to log the event into syslog?

**Comment by Habbie** [comment, marked as answer]
We already log this at level Info (which requires `loglevel=6` or up).
